### PR TITLE
ColorPicker: extending, updating the documentations

### PR DESCRIPTION
**Pull request by donlaci**
After release of the custom color formats feature in the color picker module, the documentations are updated.

### CODE REVIEW

File: hub/powertoys/color-picker.md [hunk 1]
```diff
@@ -56,17 +56,80 @@ The color in the middle represents your currently selected color from the colors
 
 To remove any color from the colors history, right click a color and select **Remove**.
 
+To export the colors history, right click a color and select **Export**. The exported values can be grouped by the colors or by the formats.
```
Comment (jaimecbernardo) [inline hub/powertoys/color-picker.md]: "can be grouped by the colors or by the formats." > can be grouped by colors or by formats.

**Review by jaimecbernardo — APPROVED** [review]
Just a nit. Other than that, LGTM!

**Comment by jaimecbernardo** [comment]
@crutkas , please review as well. Thank you!

**Review by crutkas — APPROVED** [review]

**Comment by jaimecbernardo** [comment]
Hi @mattwojo , this one is ready to go in whenever you have the time. These features are out already. (not urgent, though)
Thank you!

**Review by mattwojo — APPROVED** [review]
Made a few minor editorial updates for consistency and accessibility. We generally try to avoid words like "press" in favor of select given not everyone uses a mouse and keyboard. Sometimes this may be unavoidable in cases like "right-click", but just a general thing to keep in mind. I will also run one of the gifs through an optimization tool like ezgif.com to see if I can get the file size reduced a bit. 2MB is a bit large, the others are fine. It will work either way, just affects page load speed for folks who may not have good connections.

File: hub/powertoys/color-picker.md [hunk 2]
```diff
@@ -48,28 +48,92 @@ The copied color will be stored in your clipboard in the format that is configur
...
 - Behavior of activation shortcut
-- Format of a copied color (HEX, RGB, etc.)
+- The default format of a copied color (HEX, RGB, etc.)
 - Show the name of the color. The shown color name will be a high-level representation of the color. Example: Light Green, Green, Dark Green.
 - Order and presence of color formats in the editor
 
-![ColorPicker Settings screenshot.](../images/pt-colorpicker-settings.png)
+![ColorPicker Settings screenshot.](../images/pt-colorpicker-settings.gif)
+
+To reorder the colors, select the **...** buttom on the right and select **Move up** or **Move down**.
```
Comment (Jay-o-Way) [inline hub/powertoys/color-picker.md:73]: @donlaci @mattwojo What is a _buttom_ ?

**Review by Jay-o-Way — CHANGES_REQUESTED** [review]
Typo

File: hub/powertoys/color-picker.md (hunk 2 above)
Comment (crutkas) [reply hub/powertoys/color-picker.md:73]: good catch, https://github.com/MicrosoftDocs/windows-dev-docs/pull/4198 has fix

**Pull request merged**